src/applications/facility-locator/containers/FacilityDetail.jsx

```diff
@@ -67,12 +67,13 @@ class FacilityDetail extends Component {
         <div>
           <LocationDirectionsLink location={facility} />
         </div>
-        {phone && (
-          <p className="p1">
-            Planning to visit? Please call first as information on this page may
-            change.
-          </p>
-        )}
+        {phone &&
+          phone.main && (
```

Comment:
This looks good! The only thing I would say is if the conditional rendering gets more complex than this, we should create a separate variable on line 47.5 for it so it's easier to read :)

Reply:
for sure! thanks!